pages/examples/music/content/pages/musicpage.tsx

```diff
@@ -0,0 +1,188 @@
+  return (
+    <ScrollArea h="90vh">
+      <Box>
+        <TitleBlockComponent
+          title="Listen Now"
+          outline="Top picks for you. Updated daily."
+        />
+
+        <Box p={4} overflowX="scroll" maxW="6xl">
+          <Flex gap="4">
+            <Card variant="elevated" maxW="xs">
+              <CardHeader>
+                <Center>
+                  <Image
+                    src="https://dragon-ball-official.com/assets/img/intro/intro_1.png"
+                    maxW="xs"
+                    alt="CardImage"
+                  />
+                </Center>
+              </CardHeader>
+              <CardBody>
+                <Flex direction="column">
+                  <Text as="b" fontSize="md">
+                    YamadaUI
+                  </Text>
+                  <Text fontSize="sm">Hirotomo Yamada</Text>
+                </Flex>
+              </CardBody>
+            </Card>
+
+            <Card variant="elevated" maxW="xs">
+              <CardHeader>
+                <Center>
+                  <Image
+                    src="https://dragon-ball-official.com/assets/img/intro/intro_1.png"
+                    maxW="xs"
+                    alt="CardImage"
+                  />
+                </Center>
+              </CardHeader>
+              <CardBody>
+                <Flex direction="column">
+                  <Text as="b">YamadaUI</Text>
+                  <Text>Hirotomo Yamada</Text>
+                </Flex>
+              </CardBody>
+            </Card>
+
+            <Card variant="elevated" maxW="xs">
+              <CardHeader>
+                <Center>
+                  <Image
+                    src="https://dragon-ball-official.com/assets/img/intro/intro_1.png"
+                    maxW="xs"
+                    alt="CardImage"
+                  />
+                </Center>
+              </CardHeader>
+              <CardBody>
+                <Flex direction="column">
+                  <Text as="b">YamadaUI</Text>
+                  <Text>Hirotomo Yamada</Text>
+                </Flex>
+              </CardBody>
+            </Card>
+
+            <Card variant="elevated" maxW="xs">
+              <CardHeader>
+                <Center>
+                  <Image
+                    src="https://dragon-ball-official.com/assets/img/intro/intro_1.png"
+                    maxW="xs"
+                    alt="CardImage"
+                  />
+                </Center>
+              </CardHeader>
+              <CardBody>
+                <Flex direction="column">
+                  <Text as="b">YamadaUI</Text>
+                  <Text>Hirotomo Yamada</Text>
+                </Flex>
+              </CardBody>
+            </Card>
+          </Flex>
+        </Box>
+      </Box>
+
+      <Box pt={8}>
+        <TitleBlockComponent
+          title="Made for You"
+          outline="Your personal playlists. Updated daily."
+        />
+
+        <Box p={4} gap={4}>
+          <Flex>
```

Comment:
Make sure flex gap is not disabled

```suggestion
        <Box p={4}>
          <Flex gap="4">
```